src/main/java/de/tobiasroeser/lambdatest/proxy/TestProxy.java

```diff
@@ -135,55 +135,64 @@ public static IgnoreMethod ignoreMethod(final String name) {
 				return "Proxy[" + mkString(interfaces, " & ") + "]@" + System.identityHashCode(proxy);
 			} else {
 				final String methodSignature = methodSignature(method);
-				throw new UnsupportedOperationException("" +
-						"Unhandled call: proxy=" + proxy + ", method=" + method + ", args=" +
-						(args == null ? "null" : mkString(args, ", ")) +
-						"\nTo handle this call in the proxy delegate object, add a method with the following signature:"
-						+
-						"\n  " + methodSignature + " {\n  }" +
-						"\n");
+				final String interfaceName = method.getDeclaringClass().getSimpleName();
+
+				final String argObjects = args == null ? "" : mkString(args, ", ");
+				throw new UnsupportedOperationException(""
+						+ "Unhandled call: proxy=" + proxy + ", method=" + method + ", args=" + argObjects
+						+ "\nTo handle this call in the proxy, add the following to " + interfaceName
+						+ "\n ==> " + methodSignature + "{}"
+						+ "\n");
```

Comment:
What's the motivation to "revert" the changes made after your last merged PR #29? Looks like an overlook?